framework/transport/ieee1905_transport/include/mapf/transport/ieee1905_transport.h

```diff
@@ -154,6 +154,21 @@ class Ieee1905Transport {
     };
     friend std::ostream &operator<<(std::ostream &os, const Packet &m);
 
+    // reliable multicast neighbours map
+    struct ieee1905_neighbor {
+        sMacAddr al_mac;
+        unsigned int if_index;
+        std::chrono::steady_clock::time_point last_seen;
+    };
+
+    std::unordered_map<sMacAddr, ieee1905_neighbor> neighbors_map_;
```

Comment:
I think it makes more sense using one of the simpler containers such as unordered_map in this case. We don't expect the number of neighbors to be more than 10, and that's also exaggerating. Hence I prefer to keep the current implementation unless I'm mistaken and there could be hundreds or millions of neighbors...

Reply:
My point was: priority_queue is the simpler container for this use case. I doubt that it would be more efficient because (like most containers) it's implemented on top of std::vector.

And what I haven't made clear in my comment: there's no need to change the code now, I'm just pointing out that in the future you might consider a priority_queue for this kind of use case.